src/main/scala/org/constellation/API.scala

```diff
@@ -57,7 +56,7 @@ case class ProcessingConfig(
   maxWidth: Int = 10,
   minCheckpointFormationThreshold: Int = 50,
   numFacilitatorPeers: Int = 2,
-  randomTXPerRoundPerPeer: Int = 100,
+  randomTXPerRoundPerPeer: Int = 30,
```

Comment:
It would good idea to externalize it with hocon config

Reply:
We definitely should keep track of these in a config, I don't think this is the pr to do it in but we will in due time